### ISSUE TITLE
RSS to Slack

### ISSUE THREAD

**Issue by yoannfleurydev** [CodeursEnSeine/codeursenseine-new#39]
Les posts de meetup génère un flux RSS, ça pourrait être sympa d'avoir un post sur Slack automatiquement dès qu'un nouveau meetup est ajouté.

https://www.codeursenseine.com/meetups/rss.xml

**Comment by ntorionbearstudio** [comment]
J'ai ajouté l'application RSS sur le compte Slack de Codeurs En Seine pour qu'on ait une publication automatiquement sur le channel #communication quand il y a une mise à jour sur le flux RSS

**Comment by yoannfleurydev** [comment]
Super, merci, je pense que tu peux même mettre ca dans le #general directement comme communication est un peu polluer par tous les posts de réseaux sociaux.

**Comment by youenchene** [comment]
C'est mis en place ?

je viens d'ajouter un nouveau meetup.

**Comment by yoannfleurydev** [comment]
Oui, ça vient d'apparaitre dans le channel Slack #communication, comme c'est pas courant, je suggère de mettre ces messages dans #general directement pour avertir les membres du Slack.
